nodeup/pkg/model/kube_apiserver_test.go

```diff
@@ -31,38 +31,52 @@ func Test_KubeAPIServer_BuildFlags(t *testing.T) {
 	}{
 		{
 			kops.KubeAPIServerConfig{},
-			"--insecure-port=0 --secure-port=0",
+			"--insecure-port=0 --max-requests-inflight=0 --secure-port=0",
```

Comment:
Unfortunately I think this might actually be honored - i.e. we would actually set the limit as 0.

I actually thought that int=0 defaulted to a omitted flag, but I guess not.

I think you need to add `flag-empty:"0"`